packages/libretro/mesen-s/package.mk

```diff
@@ -0,0 +1,46 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
+#
+#  This Program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 2, or (at your option)
+#  any later version.
+#
+#  This Program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with OpenELEC.tv; see the file COPYING.  If not, write to
+#  the Free Software Foundation, 51 Franklin Street, Suite 500, Boston, MA 02110, USA.
+#  http://www.gnu.org/copyleft/gpl.html
+################################################################################
+
+PKG_NAME="mesen-s"
+PKG_VERSION="b350469"
+PKG_REV="1"
+PKG_ARCH="i386 x86_64 arm"
```

Comment:
I concur but it has been changed for [mesen package](https://github.com/libretro/Lakka-LibreELEC/blob/master/packages/libretro/mesen/package.mk). @natinusala Do you concur ? :)

Reply:
I took it out because aarch64 compilation was too long and froze my tiny build PC - since we don't support aarch64 in master I don't see any reason to put it back.

However on LE9.2 it should be put back to `any`.

Reply:
OK, we leave it like it is.